modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/StructuredContentDTOConverter.java

```diff
@@ -232,6 +232,23 @@ private Map<Locale, String> _filterDescriptionMap(
 		);
 	}
 
+	private boolean _notContainRenderedContentValue(UriInfo uriInfo) {
```

Comment:
@slnn Usually we don't just add a method called "notContainXXX". Usually the name would be "containsXXX".

Reply:
@slnn It seems there are a lot of such methods getting the parameters map from uriInfo, gets a value from the map, and then checks whether the value contains something.

Could you see if it's possible to extract a common method somewhere to simplify the code?